.changeset/eleven-guests-battle.md

```diff
@@ -0,0 +1,6 @@
+---
+"@apollo/gateway": patch
+---
+
+Move gateway post-processing errors into extensions of the response
```

Comment:
I do think in general these entries should offer a lot of explanation and context, particularly with respect to what the outcome is from the user's perspective.

In this case, references to the original change that necessitates this one and the related issue would both be helpful for building context. Additionally, I'd mention the versions affected here (2.3.0, 2.3.1), what was "broken", and the behavior we restored (and introduced).

Reply:
I'm curious as how changeset handles multi-line changelog entries, but we'll see. Happy with your edits otherwise, thanks.